src/components/__tests__/Bullet.ts

```diff
@@ -162,12 +167,14 @@ describe.skip('expansion', () => {
           - d
       `,
       }),
-      setCursor(['x', 'a', 'b', 'c']),
+      setCursor(unrankedPath),
     ])
 
-    const subthoughts = await findSubthoughts('x')
-    const bulletsOfSubthoughtsX = await findAllByLabelText(subthoughts[0], 'bullet')
-    userEvent.click(bulletsOfSubthoughtsX[0])
+    const context = contextToPath(store.getState(), unrankedPath)
```

Comment:
Get the path of the desired thought directly: `contextToPath(store.getState(), ['x', 'a'])`

Reply:
Okay I understand now thank you!

Reply:
Interesting! Okay cool. Is the `.join` still okay for this case?